src/serac/physics/materials/liquid_crystal_elastomer.hpp

```diff
@@ -179,4 +184,144 @@ struct LiquidCrystalElastomer {
   double N_b_squared_;  ///< Kuhn segment parameters.
 };
 
+/// -----------------------------------------------------------------------------
+/// -----------------------------------------------------------------------------
+/// -----------------------------------------------------------------------------
+
+/**
+ * @brief Bertoldi's liquid crystal elastomer model
+ * Paper: Li, S., Librandi, G., Yao, Y., Richard, A. J., Schneider‐Yamamura, A., Aizenberg, J., & Bertoldi, K. (2021).
+ * Controlling Liquid Crystal Orientations for Programmable Anisotropic Transformations in Cellular Microstructures.
+ * Advanced Materials, 33(42), 2105024.
+ */
+struct LiqCrystElast_Bertoldi {
+  /// this model is only intended to be used in 3D
+  static constexpr int dim = 3;
+
+  /// internal variables for the liquid crystal elastomer model
+  struct State {
+  };
+
+  /**
+   * @brief Constructor
+   *
+   * @param rho mass density of the material (in reference configuration)
+   * @param young_modulus Bulk modulus of the material
+   * @param poisson_ratio Poisson ratio of the material
+   * @param initial_order_parameter Initial value of the order parameter
+   * @param beta_parameter Parameter for degree of coupling between elastic and nematic energies
+   */
+  LiqCrystElast_Bertoldi(double rho, double young_modulus, double poisson_ratio, double initial_order_parameter,
+                         double beta_parameter)
+      : density(rho),
+        young_modulus_(young_modulus),
+        poisson_ratio_(poisson_ratio),
+        initial_order_parameter_(initial_order_parameter),
+        beta_parameter_(beta_parameter)
+  {
+    SLIC_ERROR_ROOT_IF(density <= 0.0, "Density must be positive in the LCE material model.");
+    SLIC_ERROR_ROOT_IF(young_modulus_ <= 0.0, "Bulk modulus must be positive in the LCE material model.");
+    SLIC_ERROR_ROOT_IF(poisson_ratio_ <= 0.0, "Poisson ratio must be positive in the LCE material model.");
+    SLIC_ERROR_ROOT_IF(initial_order_parameter_ <= 0.0,
+                       "Initial order parameter must be positive in the LCE material model.");
+    SLIC_ERROR_ROOT_IF(beta_parameter_ <= 0.0, "The beta parameter must be positive in the LCE material model.");
+  }
+
+  /// -------------------------------------------------------
+
+  /**
+   * @brief Material response
+   *
+   * @tparam DisplacementType number-like type for the displacement vector
+   * @tparam DispGradType number-like type for the displacement gradient tensor
+   * @tparam GammaAngleType number-like type for the orientation angle gamma
+   *
+   * @param[in,out] state A state variable object for this material. The value is updated in place.
```

Comment:
Apparently this is an open feature request on Doxygen, but should we leave unnamed variables here or remove them?

Reply:
I say remove them.